### ISSUE TITLE
[TUTORIAL] HOW TO COMPILE/USE FOR BRAINDEADS

### ISSUE THREAD

**Issue by juckthaltnicht** [swordbluesword/PalWorld-NetCrack#42]
So this is a simple step by step tutorial for how to compile in into a .dll file. 

1. Install Visual Studio 2022 (it's free) 
2. Install the C+ packages in installer NOT C# ONLY C+++ 
3. Download the repository as a ZIP
4. Unzip it 
5. Open the PalWorld-NetCrack-master folder and open the file DX11-Base.sln

6. Follow my video
- https://streamable.com/givnwn
- `$(ProjectDir);%AdditionalIncludeDirectories)` for copy and paste 
- `/bigobj` for copy and paste
- Download Xenos it's a simple dll injector (https://github.com/DarthTon/Xenos/releases/tag/2.3.2)

When you want a compiled version you can dm me on discord
DC: juckthaltnicht

WARNING I PERSONAL ONLY USE THIS ON MY SERVER USE IT ON OTHER SERVERS YOU CAN GET BANNED!

**Comment by mus6677** [comment]
![image](https://github.com/swordbluesword/PalWorld-NetCrack/assets/14856579/e1cf9667-4992-4e91-bf26-28f2ece69e7a)
If you change the build from debug to release, you don't need to change the project properties.
Can straight build the dll.

**Comment by MGJerry** [comment]
Nice guide my man, now the issues page won't be flooded with people asking how to compile. 😄 
So disappointed in today's society, where people are willing to spend time to google how to cheat, but not willing to spend the same amount of time learning how to compile the cheat. Also, Visual Studio even made it awfully clear what's wrong, you just have to read the lines above the build result.
- Cannot open include file: 'SDK.hpp' ? Just google "Visual Studio cannot open include file".
- compile with /bigobj ? Just google "compile with /bigobj"

**Comment by Hi-ImKyle** [comment]
Now the SDK is a separate project, some people will encounter a problem building the NetCrack one as Visual Studio will complain the SDK lib was build using a different compiler version. You may want to tell people to build the SDK first, THEN the NetCrack one.

**Comment by amaralkaff** [comment]
yeah sam issue any solution, im new inC++ (Now the SDK is a separate project)

**Comment by Hi-ImKyle** [comment]
> yeah sam issue any solution

> You may want to tell people to build the SDK first, THEN the NetCrack one.
